### PR TITLE
new script for dumping new SGD references with new sgdids 

### PR DESCRIPTION
**Pull request by sweng66**

### CODE REVIEW

File: crontab [hunk 1]
```diff
@@ -1,9 +1,10 @@
-SHELL=/bin/bash
+cSHELL=/bin/bash
```
Comment (valearna) [inline crontab:1]: Is this an intended change or a typo? If it's intended, what does it do?

**Review by valearna — DISMISSED** [review]
Looks good to me. Thanks @sweng66

File: crontab (hunk 1 above)
Comment (sweng66) [reply crontab:1]: oops good catch! will fix

Comment (sweng66) [reply crontab]: fixed

**Comment by sweng66** [comment]
Thanks @valearna

**Comment by sweng66** [comment]
@valearna can you reapprove this pr? thanks

**Review by valearna — APPROVED** [review]

**Pull request merged**